internal/datastore/crdb/tx_test.go

```diff
@@ -117,7 +117,7 @@ func TestTxReset(t *testing.T) {
 			require := require.New(t)
 
 			ds := b.NewDatastore(t, func(engine, uri string) datastore.Datastore {
-				ds, err := NewCRDBDatastore(
+				ds, err := newCRDBDatastore(
```

Comment:
do we not want the tests with the proxy wrapper that we will be run in practice?

Reply:
The tests use private fields on the datastore so we need to do this. No tests rely on cancellation.